Tests/SentryTests/SentryCrash/SentryCrashJSONCodec_Tests.m

```diff
@@ -463,6 +463,55 @@ - (void)testSerializeDeserializeDictionaryMultipleEntries
     XCTAssertEqualObjects(result, original, @"");
 }
 
+- (void)testSerializeDeserializeDictionaryIntKeys
+{
+    NSString *expectedJson = @"{\"1\":\"Value\",\"2\":1000,\"3\":true}";
+    NSDictionary *expected = @{ @"1" : @"Value", @"2" : @1000, @"3" : @YES };
+    NSDictionary *original = @{ @1 : @"Value", @3 : @YES, @2 : @1000 };
```

Comment:
L: I'd rename `original` to `actual` so we'd have these semantics:

>     actual - the actual value
>     expected - the expected value

Reply:
I named it like this because all the other test methods use the same terminology. I would like to keep it like this in this test class and maybe change all of them together, but not in this PR.